Marlin/src/pins/pins_AZTEEG_X5_GT.h

```diff
@@ -24,8 +24,8 @@
  * Azteeg X5 GT pin assignments
  */
 
-#ifndef TARGET_LPC1768
-  #error "Oops! Make sure you have the LPC1768 environment selected in your IDE."
+#if !defined(TARGET_LPC1768) || !defined(LPC1769)
```

Comment:
Since `LPC1769` will never be defined without `TARGET_LPC1768` this can just be:
```cpp
#ifndef LPC1769
```